R/gsDesign2-package.R

```diff
@@ -21,9 +21,9 @@
 #' @importFrom dplyr "%>%" all_of arrange desc filter full_join group_by lead
 #'   left_join mutate one_of rename row_number select summarize ungroup
 #' @importFrom gsDesign gsDesign sfLDOF
-#' @importFrom stats pnorm qnorm setNames stepfun uniroot
+#' @importFrom stats pnorm qnorm setNames uniroot
 #' @importFrom tibble tibble
-#' @importFrom utils tail
+#' @import utils
```

Comment:
This would be a good discussion topic for a future group meeting. I prefer to explicitly list each imported function like we currently do for {data.table} and {dplyr}.

Reply:
For base packages, usually I fully import them. For 3rd party packages, usually I import functions explicitly. I'm okay with selective imports for base packages, too. In fact, the stats package has to be imported selectively to avoid conflicts between `stats::filter` and `dplyr::filter`.